### ISSUE TITLE
Device icon

### ISSUE THREAD

**Issue by Vaifranz** [DoctorNSA/homebridge-musiccast-tv#20]
Hi, thank you for your work, really great, I managed to add some Yamaha MultiCast devices, also WXAD-10 in Link with a HTR-4096, I only have one request, would it be possible to change the icon of the accessory? I have an iPhone XS Max with iOS 15.2.1 and it displays a house instead of a sound device, thanks.
![IMG_E1A830E46AF4-1](https://user-images.githubusercontent.com/33608948/150650256-ce1003a5-bc55-4bcd-867d-5fc511e1e877.jpeg)

**Comment by aangert** [comment]
Yes, this used to work, unfortunately it broke a few iOS versions ago. I will reimplement it in the future

**Comment by Vaifranz** [comment]
Thanks in advance, I hope you will be able to implement it as soon as possible, this plugin is very convenient. Thanks again for your work.

**Comment by benchristian88** [comment]
keen on this too.  other that the icon, this works perfect for multizone yamaha amp, with multiple inputs.
